consensus/types/src/blob_sidecar.rs

```diff
@@ -154,7 +162,7 @@ impl<T: EthSpec> BlobSidecar<T> {
             blob,
             kzg_commitment: commitment,
             kzg_proof: proof,
-            ..Default::default()
+            ..Self::empty()
```

Comment:
Cool trick! didn't know we could use arbitrary functions to fill remaining fields.